config/config-builder/src/swarm_config.rs

```diff
@@ -173,6 +173,10 @@ impl SwarmConfig {
             network.listen_address = utils::get_available_port_in_multiaddr(is_ipv4);
             network.advertised_address = network.listen_address.clone();
             add_peer(network, &mut network_peers, &mut seed_peers);
+            // @TODO the validator scripts expect this to be in this format. This restriction
```

Comment:
This commit is a bit unfortunate. It would be nice if we didn't have to regenerate all of these things. This probably makes this a breaking change too

Reply:
The actual breaking change is somewhat of a meta breaking change. Specifically the use of the RNG is not in the same order, hence keys are generated differently. But the actual config itself remains the same.